src/solver/hydro/management/daily.cpp

```diff
@@ -546,9 +561,7 @@ void HydroManagement::prepareDailyOptimalGenerations(Solver::Variable::State& st
                                                      uint y,
                                                      Antares::Data::Area::ScratchMap& scratchmap)
 {
-    areas_.each(
-      [&](Data::Area& area) {
-          prepareDailyOptimalGenerations(state, area, y, scratchmap);
-          });
+    areas_.each([&](Data::Area& area)
```

Comment:
Here the first part of the lambda is not rejected to the next line. In some other files, it is. 
Strange.

Reply:
Agreed that the behaviour is strange, but it does not seem to be possible to change it unfortunately (except by disabling completely single line lambdas).